src/cryptoadvance/specter/rpc_cache.py

```diff
@@ -0,0 +1,36 @@
+from .rpc import BitcoinCLI
+from .corecache import CoreCache
+
+class BitcoinCLICached:
+    def __init__(self, user="", passwd="", host="127.0.0.1", port=8332, protocol="http", path="", timeout=30, cli=None, **kwargs):
```

Comment:
This is accepting two kind of constructors for that class. This is not very clear from the code. Can you rewrite this as described here: https://stackoverflow.com/questions/141545/how-to-overload-init-method-based-on-argument-type ?

Reply:
Added, let me know if it's good now.